### ISSUE TITLE
Plugin not loading correctly?

### ISSUE THREAD

**Issue by banksSAMP** [oscar-broman/SKY#9]
So I'm starting up a dm server for myself(and a couple of friends) and I felt that your damage system is a must-have. I downloaded & installed the include aswell as the SKY plugin, but when I compiled my gamemode it gave me this error:
C:\Users\John\Desktop\sampskit\APGW_admintest\pawno\include\weapon-config.inc(62) : fatal error 111: user error: The SKY plugin is required, get it here: github.com/oscar-broman/sky
I browsed about on the replies on the sampforums release thread, I found and tried the most recent compiled windows versions but to no avail as they all gave me either the error message above or another one about me having an unknown windows server version or something like that.

I struggled for quite a while, installing the wrong Visual Studio and whatnot before finally finding the correct version(sounds like a simple task yet I still managed to fuck up). Downloaded the most recent source code thing and compiled it, put the .dll inside my plugins folder only to get the error message listed at the top. The server is running 0.3.7 R2. Let me know if there is any other information you need.

So long story short I guess I need some guidance on how to fix this.

Please note that I am relatively new to scripting in general so I'd greatly appreciate if you could explain this in the simplest way possible.

Banks

**Comment by oscar-broman** [comment]
Hey,
This release works: https://github.com/oscar-broman/SKY/releases

There are a few minor fixes I didn't push to Git just yet (I forgot), that's why you couldn't compile it properly for the latest version.

**Comment by banksSAMP** [comment]
Damn. Amazing response time, I compiled the new dll and the original problem seemed to have been fixed , I get the following in the console when I boot up the server:
"Loading plugin: SKY"
"Loaded."
However when I add in the weapon-config(#include <weapon-config>)
It just gives me the same ol'
"The SKY plugin is required, get it here:" error even though I know its there. :S

Banks

**Comment by oscar-broman** [comment]
How about this response time?

You need to put `SKY.inc` among your include files.

**Comment by banksSAMP** [comment]
Haha 2fast4me man! :D
Slowly getting there,  the following pops up when I compile my gamemode:
http://pastebin.com/74rmaa2p

**Comment by oscar-broman** [comment]
Update your include files.

Create issues on oscar-broman/samp-weapon-config if you need further help.
